app/api/sessions.py

```diff
@@ -59,8 +60,16 @@ def before_post(self, args, kwargs, data):
         ):
             raise ForbiddenError({'pointer': ''}, "Sessions are disabled for this Event")
 
+        
+        custom_fields = db.session.query(CustomForms).filter_by(event_id = data['event']).all()
+        for i in custom_fields:
+            if(i.field_identifier == "track" and i.is_required == True):
+                excluded = []
+            elif(i.field_identifier == "track" and i.is_required == False):
+                excluded = ['track']
```

Comment:
Shouldn't be needed now

Reply:
only removing track from required relation is not working.

Reply:
What's the error?

Reply:
sorry, I had made a mistake. It's working corretly.